crates/iceberg/src/lib.rs

```diff
@@ -21,7 +21,9 @@
 //!
 //! ## Scan A Table
 //!
-//! ```rust, no_run
+//! ```rust, ignore
```

Comment:
@liurenjie1024 wdyt of using `ignore` here for now? i didnt want to remove all the docs